### ISSUE TITLE
Use of Front Door for Inbound Traffic to App Service

### ISSUE THREAD

**Issue by nanddeepn** [OfficeDev/microsoft-teams-apps-company-communicator#550]
Hello,
As we have a Front Door, we want to use it for **Access restriction** for **Inbound traffic** in **App Service**.

On **App Service**, under **Settings**, click **Networking**.

![image](https://user-images.githubusercontent.com/8310319/141484490-726ac85d-bc22-4d4f-8733-59e5aaa94e23.png)

Add a rule for Front Door.
![image](https://user-images.githubusercontent.com/8310319/141483786-6ace798c-7917-459a-8bed-7c2f681c45a3.png)

Now, when we open the Company Communicator App in MS Teams, we start to see CORS errors.

`Access to XMLHttpRequest at 'https://communicator.azurewebsites.net/api/draftnotifications' (redirected from 'https://communicator.azurefd.net/api/draftnotifications') from origin 'https://communicator.azurefd.net' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

![image](https://user-images.githubusercontent.com/8310319/141485319-6482cf66-2664-4f51-a698-ae729bfba3aa.png)

Can someone please suggest?
Thank you!

**Comment by Porkai-Pandian** [comment]
Hi @nanddeepn,

Thanks for raising the query, we'll simulate this scenario using Azure Front door and explore the issue which you're facing and get back to you with our findings.

**Comment by SankarMicrosoft** [comment]
Please let us know if you are still facing this issue and require assistance and we will be happy to discuss. If no further action is needed at moment and no response is received, we will close out the issue in 3 business days. You may also always open a new issue or respond here if something does come back up.

**Comment by nanddeepn** [comment]
Hi @SankarMicrosoft 
You may close this issue.

**Comment by sponlinedev** [comment]
how to solve this issue , we are also facing similar issue

**Comment by nanddeepn** [comment]
Hi @sponlinedev 
We had used the certificates to get across the CORS issue.
